tests/more-tests/externals_multiple_files/test_externals_multiple_files.mdl

```diff
@@ -0,0 +1,116 @@
+{UTF-8}

```

Comment:
Please include these tests in the test-models repo and keep the more-test folder only for tests that cannot be fully integrated

Reply:
the test is not very relevant....it's just to be able to test the option to include/exclude from serialization external objects from a separate file. So it's quite useless for all other projects.